### PR TITLE
Updated Wideband simulation scripts

### PR DESCRIPTION
**Pull request by ehrlich-uva**
This PR has no impact on other simulations outside of the Wideband CRV simulations

### CODE REVIEW

**Comment by ehrlich-uva** [comment]
@kutschke Could you please add @oksuzian as a reviewer?

File: Mu2eG4/geom/geom_Wideband4modules_v3.txt [hunk 1]
```diff
@@ -0,0 +1,536 @@
...
+int crs.FEBBoxesAtSide1W1  = 0;
+int crs.FEBBoxesAtSide1W2  = 0;
+int crs.FEBBoxesAtSide1W3  = 0;
+int crs.FEBBoxesAtSide1W4  = 0;
+
+//bool g4.doSurfaceCheck    = true;
+int  g4.nSurfaceCheckPointsPercmsq =   1;
+int  g4.minSurfaceCheckPoints      = 100; // per volume
+int  g4.maxSurfaceCheckPoints = 10000000; // per volume
+
```
Comment (kutschke) [inline Mu2eG4/geom/geom_Wideband4modules_v3.txt:96]: I see that you have the long list of changing material to G4_AIR in ./Mu2eG4/geom/geom_common_trackerVST.txt .

Are the lists identical?  If so please make one file that has the content and #include it where needed.

If the lists are nearly identical, please put the common parts in one file and #include it.

**Review by kutschke — CHANGES_REQUESTED** [review]
I see that you made changes to CRVResponse/src/SConscript .  Please make the matching changes in  CRVResponse/CMakeLists.txt .  We are developing support for cmake builds.

File: CRVResponse/test/calibration.fcl [hunk 2]
```diff
@@ -1,103 +1,75 @@
...
+                     "drop *CrvSiPMCharges*_*_*_*",
+                     "drop *CrvDigiMC*_*_*_*"]
+  }
+}
 
-# g4 config
-physics.producers.g4run.TrajectoryControl: @local::mu2eg4NoTrajectories
-physics.producers.g4run.inputs.primaryTag : cosmicTimeOffset
-physics.producers.g4run.Mu2eG4CommonCut: @local::Cosmic.Mu2eG4CommonCutCosmicNoField
-physics.producers.g4run.physics.minRangeCut : 1.0 // mm
```
Comment (oksuzian) [inline CRVResponse/test/calibration.fcl:65]: I would keep this cut. It makes a significant impact on the file size.

File: CRVResponse/test/calibration.fcl [hunk 3]
```diff
@@ -1,103 +1,75 @@
...
 #include "Production/JobConfig/digitize/prolog.fcl"
 #include "Production/JobConfig/cosmic/prolog.fcl"
-#include "Production/JobConfig/reco/prolog.fcl"
 
-process_name : cosmicSimReco
+process_name : crvCalibSim
 
 source : {
   module_type :EmptyEvent
   firstRun: 1200
```
Comment (oksuzian) [inline CRVResponse/test/calibration.fcl:15]: That's the value that falls into the range for MDC2020.
Is that the intent?

File: CRVResponse/src/CrvWidebandTest_module.cc [hunk 4]
```diff
@@ -449,17 +449,20 @@ namespace mu2e
...
-          uint16_t feb            = onlineChannel.FEB();
-          uint16_t febChannel     = onlineChannel.FEBchannel();
-
-          int index=feb*CRVId::nChanPerFEB+febChannel;
-          _recoPEs[index]   = recoPEs;
-          _recoTime[index]  = recoTime;
-          _fitStatus[index] = fitStatus;
-          _depositedEnergy[index] = depositedEnergy;
-          _histPEs[index]->Fill(recoPEs);
+          if(iSectorType==0)
```
Comment (oksuzian) [inline CRVResponse/src/CrvWidebandTest_module.cc:452]: Please add the comment: "iSectorType==0 performs a track fit using all modules in the stack"

**Review by oksuzian — APPROVED** [review]
Looks good. Please address the choice of the run number.

**Comment by kutschke** [comment]
@ehrlich-uva a gentle reminder that there are several requests in the review so far.

File: Mu2eG4/geom/geom_Wideband4modules_v3.txt (hunk 1 above)
Comment (ehrlich-uva) [reply Mu2eG4/geom/geom_Wideband4modules_v3.txt:96]: I didn't realize that geom_common_trackerVST.txt tried to do something similar (turn the concrete into air). I left the dirt unchanged in geom_Wideband4modules_v3.txt, while geom_common_trackerVST.txt also removed the dirt and probably other things as well.

Comment (ehrlich-uva) [reply Mu2eG4/geom/geom_Wideband4modules_v3.txt:96]: Yes, I can update the CMakeList.txt

File: CRVResponse/src/CrvWidebandTest_module.cc (hunk 4 above)
Comment (ehrlich-uva) [reply CRVResponse/src/CrvWidebandTest_module.cc:452]: I will make the change.

File: CRVResponse/test/calibration.fcl (hunk 3 above)
Comment (ehrlich-uva) [reply CRVResponse/test/calibration.fcl]: Yes, this calibration script is only used for testing purposes and won't be used in any production. I wanted to have a run number (I just changed it to 1210) close other run numbers for which we already have calibration constants in the database. The purpose was to test my GUI to show historic calibration constants and compare them with newly calculated calibration constants to automatically check, if these the calibration succeeded.

File: CRVResponse/test/calibration.fcl (hunk 2 above)
Comment (ehrlich-uva) [reply CRVResponse/test/calibration.fcl:65]: True. I will bring them back.

**Comment by ehrlich-uva** [comment]
@kutschke I'm working on it.

File: Mu2eG4/geom/geom_Wideband4modules_v3.txt (hunk 1 above)
Comment (ehrlich-uva) [reply Mu2eG4/geom/geom_Wideband4modules_v3.txt:96]: I removed the change in SConscript, because it is no longer needed. I only used it for a test.

**Comment by kutschke** [comment]
> @kutschke I'm working on it.

Let me know when you are done and I will rerun the CI.

**Comment by ehrlich-uva** [comment]
I pushed the requested changes.

**Comment by kutschke** [comment]
@FNALbuild run build test

File: Mu2eG4/geom/geom_Wideband4modules_v3.txt (hunk 1 above)
Comment (kutschke) [reply Mu2eG4/geom/geom_Wideband4modules_v3.txt:96]: OK.

**Review by kutschke — APPROVED** [review]
Thanks for the explanations.

**Pull request merged**